cacli/cmd/cancel.go

```diff
@@ -22,19 +22,17 @@ THE SOFTWARE.
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/cloud-annotations/training/cacli/cmd/cancel"
 	"github.com/spf13/cobra"
 )
 
 // cancelCmd represents the cancel command
 var cancelCmd = &cobra.Command{
 	Use:   "cancel <model-id>",
-	Short: "Coming soon",
-	Long:  `Coming soon.`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("This command is not yet available.")
-	},
+	Short: "Cancel a training that is either pending or running",
+	Long: `Cancel a training that is either pending or running. 
+	You can not cancel a training that is complete or already canceled of failed.`,
```

Comment:
```
	Long: `Cancel a training run that is either running or pending. You can not cancel a
training run that is complete, canceled or failed.

Basic Example:
  cacli cancel MODEL-ID`,
```

Also make sure the string is flush to one side to avoid random spaces when logged

Reply:
will fix